apps/prelaunch/models/base.py

```diff
@@ -139,8 +139,6 @@ def notification_message(self):
 
     def send_notifications(self):
         if htk_setting('HTK_SLACK_NOTIFICATIONS_ENABLED'):
-            from htk.utils.notifications import slack_notify
```

Comment:
If imports were done previously at the function level, I suggest leaving them there for now.

While it's generally better to import at the top of a file, there is a risk that it could have been done this way to avoid a circular dependency import. Or, it could have just been oversight. But we don't have enough context from just looking at this code.

Reply:
Sounds good. Noted this to remember. Thanks.